__tests__/intergration.tests.js

```
@@ -81,196 +81,168 @@ describe("/api/articles", () => {
         }
       });
   });
-  test("GET:200 if optinal query for topics is present filter artiles by topic, otherwise returnin all articles", ()=>{
+
+  test("GET:200 if optinal query for topics is present filter artiles by topic, otherwise returnin all articles", () => {
     return request(app)
       .get("/api/articles?topic=mitch")
       .expect(200)
       .then((response) => {
-        
-        expect(response.body.articles.length).toBe(4)
-      })
-  })
-  test("GET:200 filtered articles should all have correct topic property to match the filter", ()=>{
+        expect(response.body.articles.length).toBe(4);
+      });
+  });
+
+  test("GET:200 filtered articles should all have correct topic property to match the filter", () => {
     return request(app)
       .get("/api/articles?topic=mitch")
       .expect(200)
       .then((response) => {
-        response.body.articles.forEach((article)=>{
-          expect(article.topic).toBe("mitch")
+        response.body.articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
 
-        })
-      })
-  })
-  test("GET:200 should return empty array when filtertopic is valid but does not exist", ()=>{
+  test("GET:200 should return empty array when filtertopic is valid but does not exist", () => {
```

I'd argue that a non existent filter topic should throw an error, however a topic that does exist yet has no articles should be 200 OK
eg. articles?topic=paper   -> 200 OK.  (this exists with no topics)
      articles?topic=kungfu -> 404 NOT FOUND (this topic doesn't exist)